src/main/kotlin/pet/project/app/service/BookService.kt

```diff
@@ -0,0 +1,44 @@
+package pet.project.app.service
+
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import pet.project.app.repository.BookRepository
+import pet.project.app.model.Book
+
+@Service
+class BookService(val bookRepository: BookRepository) {
+
+    fun create(book: Book) : Book {
+        return bookRepository.save(book)
+    }
+
+    fun getById(bookId: String): Book {
+        return bookRepository.findByIdOrNull(bookId) ?: throw Exception("No book with id = $bookId")
```

Comment:
```suggestion
        return bookRepository.findByIdOrNull(bookId) ?: throw BookNotFoundException("No book with id = $bookId")
```
Prefer using custom exceptions

Reply:
Fixed: Implemented custom exceptions